### PR TITLE
Update 1203699034.rst

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/Exceptions/1203699034.rst [hunk 1]
```diff
@@ -44,3 +44,7 @@ not removed the cache files in typo3conf.
 
 To do so simply remove these files go to your TYPO3 Installation and in
 the shell type `rm typo3conf/temp_CACHED_\*` and reload the page.
+
+This exception is raised if you upgraded from TYPO3 9.5 to 10.4 and then immediately to 11 .
+Solution: Delete your cookies.
```
Comment (mbrodala) [inline Documentation/Exceptions/1203699034.rst:49]: Can you add more details to this claim? The relation to caching is unclear just from reading.

Comment (franzholz) [reply Documentation/Exceptions/1203699034.rst:49]: This is not a solution with caching but with cookies.

Comment (mbrodala) [reply Documentation/Exceptions/1203699034.rst:49]: Sorry, I don't get it. Please add more details to explain the relation of these two topics.

**Review by mbrodala — CHANGES_REQUESTED** [review]
Please add more details, the relation between a broken cache and cookies (BTW which? BE? FE?) is still unclear.

**Comment by franzholz** [comment]
I could not log into the TYPO3 backend after the upgrade with the Install Tool.
It did not help to remove all temporary files and clear all caches.

**Comment by mbrodala** [comment]
Let's discuss this in the TYPO3 Slack first and/or create a ticket describing your issue. In any case it does not sound related to the exception mentioned here so I'll close this for now.

**Pull request closed without merging**